lms/djangoapps/labster_course_license/views.py

```diff
@@ -58,6 +60,36 @@ def license_handler(request, course, ccx=None):
     return HttpResponseBadRequest('Only GET and POST methods are supported.')
 
 
+class LicensedSimulationsUpdateView(APIView):
+    """
+    Labster License update handlers.
+    """
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request, license_code):
+        """
+        Update course license with new simulations list.
+        """
+        if 'licensed_simulations' not in request.data:
+            return Response(
+                {"error": "`licensed_simulations` parameter is required."},
+                status=status.HTTP_400_BAD_REQUEST
+            )
+
+        try:
+            course_license = CourseLicense.objects.get(license_code=license_code)
+        except CourseLicense.DoesNotExist:
+            return Response(
+                {"error": "License %s not found." % license_code},
+                status=status.HTTP_400_BAD_REQUEST
+            )
```

Comment:
It's common case that API has much more licenses than edX, so does it make sense to repsonse with 400 error?